photon-client/src/main/scala/com/linkedin/photon/ml/estimators/GameEstimatorEvaluationFunction.scala

```diff
@@ -46,11 +53,16 @@ class GameEstimatorEvaluationFunction(
   /**
    * Performs the evaluation.
    *
-   * @param hyperParameters The vector of hyperparameter values under which to evaluate the function
+   * @param candidate The candidate vector of hyperparameter with values in [0, 1]
    * @return A tuple of the evaluated value and the original output from the inner estimator
    */
-  override def apply(hyperParameters: DenseVector[Double]): (Double, GameResult) = {
-    val newConfiguration = vectorToConfiguration(hyperParameters)
+  override def apply(candidate: DenseVector[Double]): (Double, GameResult) = {
+
+    // Inverse-scale candidate
+    candidate *= (range.end - range.start)
+    candidate += range.start
```

Comment:
Does this mutate the candidate vector in-place? If so, we should probably make a copy first, so that the caller doesn't see any side-effects of this call.